braindecode/datautil/serialization.py

```diff
@@ -146,6 +169,8 @@ def load_concat_dataset(path, preload, ids_to_load=None, target_name=None,
     -------
     concat_dataset: BaseConcatDataset of BaseDatasets or WindowsDatasets
     """
+    path = Path(path)
```

Comment:
I would allow in the parameters to pass `pathlib.Path` also as path parameter.

Reply:
Did this! :)